src/main/client/scene/cheese-sprite.ts

```diff
@@ -54,6 +54,8 @@ export class CheeseSprite extends Phaser.GameObjects.Container {
 
    constructor(private readonly options: CheeseOptions) {
       super(options.scene, options.position.x, options.position.y);
+      this.x -= this.width / 2;
```

Comment:
what does this do?

perhaps its best to do this.setOrigin() to achive this

Reply:
If it does the same, sure. I wanted to move the sprite so the model position will be the center.